### ISSUE TITLE
Robot playback motion

### ISSUE THREAD

**Issue by garimellagowtham** [dorna-robotics/dorna#12]
Hello @smhty,
It is often cumbersome to specify exact coordinates and path the robot end effector has to take. Usually, there is a playback option for robotic arms that allows the user to cut off the power to motors to record the motion and play it back. I am wondering if there is some way of achieving this on Dorna. Is there an option to power off the motors while still allowing the user to record the position of the arm?

Thanks,
Gowtham.

**Comment by smhty** [comment]
Hi @garimellagowtham 
Unfortunately not. That requires, some kind of feedback and encoder, and Dorna does not offer any feedback.

**Comment by garimellagowtham** [comment]
Is there a way to turn the arm on/off using the API? I can find external ways to get feedback such as using markers etc. I would still need to turn the robot on/off?

On a side note, I am wondering how homing works if there is no feedback?

**Comment by smhty** [comment]
@garimellagowtham 
You do not have control over the power supply through the API.
But you can still use power management tools that G2core provides to enable and disable motors:
https://github.com/synthetos/g2/wiki/Power-Managemen

For example, the g2core command `{1pm:0}` disables motor `1`

The homing sensors are there for homing, and setting each joint to a known position.
After the homing process is done, the position of each joint is related to the steps of the associated motors.
So basically, the robot knows its position only when the motors are on, and by turning off the motors, there is no way to recover the position. That's why homing is required whenever you turn on the robot.
